Nef_3/include/CGAL/Nef_3/Halfedge.h

```diff
@@ -21,6 +21,7 @@
 
 #include <string>
 #include <sstream>
+#include <CGAL/IO/io.h>
```

Comment:
If you have to add it here, should it not have disappeared on the left side of the diff.    Or does it just already included indirectly? Then even adding it now makes the code cleaner.